infra/token-transfer-server/src/controllers/user.js

```diff
@@ -93,4 +95,12 @@ router.post(
   })
 )
 
+router.get('/user-stats', async (req, res) => {
+  return res.status(200).send({
+    userCount: await Lockup.count({
```

Comment:
Does this count the sum total of locked tokens or the # of distinct people who locked up?  I believe we want the former for the dashboard.

Reply:
@joshfraser Have added that. Also, noticed that I didn't mention `col` along with `distinct`. Have fixed it too.

Reply:
There is also an additional confirmed column that will need to be true.

Reply:
Thanks @tomlinton. Have updated it.